libstorage/drivers/storage/ebs/executor/ebs_executor.go

```diff
@@ -165,11 +166,34 @@ func (d *driver) LocalDevices(
 			continue
 		}
 		devName := fields[3]
+
+		devPath := path.Join("/dev/", devName)
+		devAlias := devPath
+
+		// NVMe support
+		if strings.Contains(devName, "nvme") {
+			// find the EBS device name that we *think* we will mount the device as (nvme ignore this)
+			if out, err := exec.Command("/usr/sbin/nvme", "id-ctrl", "--raw-binary", devPath).Output(); err == nil {
+				// read the binary output slice and trim it
+				dev := strings.TrimSpace(string(out[3072:3104]))
+				// if the result contains a /dev/ then we got a match
+				if strings.Contains(dev, "/dev/") {
+					log.Debugf("Found symlink for '%s' -> '%s'", devName, dev)
```

Comment:
Please use the context logger, not `log` directly. Also please use the structured logging as is used everywhere else in RR.